specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/WebApps.json

```diff
@@ -20581,6 +20804,59 @@
         }
       }
     },
+    "CsmPublishingCredentialsPoliciesCollection": {
+      "description": "Publishing Credentials Policies collection.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "./CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "CsmPublishingCredentialsPoliciesCollection resource specific properties",
+          "required": [
+            "ftp",
+            "scm"
+          ],
+          "properties": {
+            "ftp": {
+              "$ref": "#/definitions/CsmPublishingCredentialsPoliciesEntity",
+              "description": "Whether FTP is allowed."
+            },
+            "scm": {
+              "$ref": "#/definitions/CsmPublishingCredentialsPoliciesEntity",
+              "description": "Whether Scm Basic Auth is allowed."
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "CsmPublishingCredentialsPoliciesEntity": {
+      "description": "Publishing Credentials Policies parameters.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "./CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
```

Comment:
Do you want to add ProxyResource properties inside each CsmPublishingCredentialsPoliciesEntity? Looks a bug to me,

Reply:
This resource is rightfully a ProxyOnly resources as it is not meant to be tracked by ARM. It does not actually map to a new resource but rather provides an API for customers to update a configuration and then configure RBAC on this particular config.